docs/permission/plugin-authors/02-adding-a-basic-permission-check.md

```diff
@@ -1,55 +1,41 @@
 ---
-id: authorize-the-create-endpoint
-title: Authorize the create endpoint
-description: Authorize the create endpoint
+id: 02-adding-a-basic-permission-check
+title: 2. Adding a basic permission check
+description: Explains how to add a basic permission check to a Backstage plugin
 ---
 
-The first step we need to do in order to authorize the create endpoint, is to create a new `permission` inside our backend plugin.
+If the outcome of a permission check doesn't need to change for different [resources](../concepts#resources-and-rules), you can use a _basic permission check_. For this kind of check, we simply need to define a [permission](../concepts#resources-and-rules), and call `authorize` with it.
 
-// TODO(vinzscam): add links to the core concepts
+For this tutorial, we'll use a basic permission check to authorize the `create` endpoint in our todo-backend. This will allow Backstage integrators to control whether each of their users is authorized to create todos by adjusting their [permission policy](../concepts#policy).
+
+We'll start by creating a new permission, and then we'll use the permission api to call `authorize` with it during todo creation.
+
+## Setup
 
 Install the following module:
 
 ```
 $ yarn workspace @internal/plugin-todo-list-backend add @backstage/plugin-permission-common
 ```
 
-Let's create a new `permissions.ts` file under `plugins/todo-list-backend/src/service/permissions.ts` with the following content:
+## Creating a new permission
```

Comment:
Added titles throughout since it means we get a table of contents on the right-hand-side of the page.